mkdocs.yml

```diff
@@ -218,6 +219,7 @@ nav:
           - Backporting: community/maintainer/backporting.md
           - Help Wanted: community/maintainer/help-wanted.md
           - Triage: community/maintainer/triage.md
+
 theme:
```

Comment:
Can we add a theme switcher?

```diff --git a/mkdocs.yml b/mkdocs.yml
index be26a3899..fc93573e7 100644
--- a/mkdocs.yml
+++ b/mkdocs.yml
@@ -236,6 +236,20 @@ theme:
     - content.code.copy
   font:
     text: Inter
+  palette:
+    # Palette toggle for light mode
+    - media: "(prefers-color-scheme: light)"
+      scheme: material
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+
+    # Palette toggle for dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
 
 markdown_extensions:
   - pymdownx.highlight
~
~
~
```

Reply:
I don't mind adding, but it would have to work with the rest of the theme, and currently doesn't look good with the link and highlight colors. let's merge this PR as an initial step and later feel free to add a PR that adds dark mode